packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -47,12 +108,21 @@ interface GetDialogueMessageProps {
   newMessageContents: string
 }
 
-interface NotificationType {
+export interface NotificationType {
   name: string
   userSettingField: keyof DbUser|null
   allowedChannels?: NotificationChannelOption[],
   getMessage: (args: {documentType: NotificationDocument|null, documentId: string|null, extraData?: Record<string,any>})=>Promise<string>
-  getIcon: ()=>React.ReactNode
+  getIcon: () => ReactNode,
+  Display?: FC<{
```

Comment:
The `Display` functions allow us to have interactive versions instead of the plain text returned by `getMessage`